### DISCUSSION TITLE
Renovate is incorrectly using the github-tags host type for the 'Go' datasource

### DISCUSSION THREAD

**Discussion by uhthomas** [renovatebot/renovate#17942] [Request Help]
Hi, I raised this issue (https://github.com/renovatebot/renovate/issues/17923) but wanted to open a discussion as it may be a more appropriate medium. Does anyone have any advice on whether something is misconfigured or if this is an actual bug in Renovate?

**Comment by rarkins** [comment]
Renovate "understands" a few module host types but then falls back to GitHub. I think what would be better would be for Renovate to fall back to `git-refs` so that any git source could be supported.

**Reply by uhthomas** [reply]
I see, how easy would it be to change this? I don't believe we can use Renovate effectively without this change.

**Reply by uhthomas** [reply]
Is it possible to configure it for a something different? The underlying Git server is Bitbucket Server, so it would probably be better to use something designed for Bitbucket rather than a generic git refs.

**Reply by rarkins** [reply]
Please try to create a minimal public reproduction. Presumably you can't link to your Bitbucket Server, but hopefully you can find a public package which needs to use raw git refs as the fallback instead of GitHub. A `go.mod` with single dependency should be enough, does not even need a `go.sum`.
